src/main/java/com/whalespottingjava/controllers/SightingController.java

```diff
@@ -55,12 +54,6 @@ public String submitSighting(@ModelAttribute Sighting sighting, Model model) {
     return "add_sighting_confirmation";
   }
 
-  @GetMapping("/sightings")
-  public String getAllSightings(Model model) {
-    model.addAttribute("sightings", sightingService.getAllSightings());
-    return "sighting_test";
-  }
-
   @GetMapping("/sightings/approved")
```

Comment:
Do we still need this endpoint if /sightings shows the exact same thing?

Reply:
Method has been removed; permitted by original creator.